apps/prairielearn/src/middlewares/clientFingerprint.ts

```diff
@@ -0,0 +1,43 @@
+import asyncHandler = require('express-async-handler');
+import * as sqldb from '@prairielearn/postgres';
+import { IdSchema } from '../lib/db-types';
+
+const sql = sqldb.loadSqlEquiv(__filename);
+
+export default asyncHandler(async (req, res, next) => {
+  const userSessionId = await sqldb.queryOptionalRow(
+    sql.select_session_id,
+    { session_id: req.session.id, user_id: res.locals.user.user_id },
```

Comment:
It is not. This was being used in a previous iteration of the query but has since been changed. Removed `user_id` parameter in commit [ee8e1f7](https://github.com/PrairieLearn/PrairieLearn/pull/8798/commits/ee8e1f776c4cb27f23ead33447cf43c21acd5ab5).